sklearn/tree/_tree.pyx

```diff
@@ -366,22 +507,25 @@ cdef class ClassificationCriterion(Criterion):
 
 
 cdef class Entropy(ClassificationCriterion):
-    """Cross Entropy impurity criteria.
+    """A class representing the Cross Entropy impurity criteria. 
```

Comment:
We don't need stuff like "A class representing the" in docstring of classes. This is true for any `class` declaration and therefore redundant.